docs/pages/choose-an-edition/migrate-to-cloud.mdx

```diff
@@ -0,0 +1,324 @@
+your Teleport Enterprise (self-hosted) deployment to Teleport Enterprise
+(cloud-hosted) successfully while ensuring minimal disruption to your
+operations.
+
+## How it works
+
+While in a cloud-hosted Enterprise Teleport account, Teleport manages the Auth
+Service and Proxy Service for you, you need to migrate dynamic resources and
+Teleport services yourself. 
+
+To migrate a self-hosted Teleport Enterprise cluster to a cloud-hosted Teleport
+Enterprise cluster:
+
+1. Set up a separate cloud-hosted Teleport account.
+1. Retrieve dynamic Teleport resources from the Auth Service backend on the
+   self-hosted cluster and apply them against the Auth Service backed on the
+   cloud-hosted cluster.
+1. Reconfigure Teleport agents and plugins to connect to the cloud-hosted
+   Teleport cluster.
+1. Verify that the migration has succeeded.
+
+## Prerequisites
+
+- An existing Teleport Enterprise (self-hosted) cluster.
+- The `tsh` and `tctl` client tools. This guide assumes that you are using
+  `tctl` to manage dynamic resources, but it is also possible to use [Teleport
+  Terraform provider](./management/dynamic-resources/terraform-provider.mdx) and
+  [Kubernetes
+  operator](./management/dynamic-resources/teleport-operator-standalone.mdx), in
+  addition to custom scripts that use the [Teleport API](./api/introduction.mdx)
+  to manage the Teleport Auth Service backend.
+
+## Step 1/4. Deploy your Teleport Enterprise (cloud-hosted) cluster
+
+1. Determine the `teleport.sh` subdomain you would like to use for your new
+   Teleport Enterprise (cloud-hosted) account. 
+
+1. If your existing Teleport Enterprise (self-hosted) license dashboard is
+   already using your desired subdomain, you can contact Teleport Support to
+   free up the domain for re-use.
+
+   [Reach out](https://goteleport.com/signup/enterprise/) to your Account
+   Management team to set up your Teleport Enterprise (cloud-hosted) tenant.
+
+1. Ensure you are running Teleport Enterprise agents with versions that are
+   lower than the Teleport Enterprise (cloud-hosted) tenant version. To check
+   the versions of your Teleport Enterprise agents, you can use the `tctl`
+   command to list the inventory of connected agents and their version:
+
+   ```code  
+   $ tctl inventory ls --older-than=<version>
+   ```
+
+Validate connectivity to both the newly deployed Teleport Enterprise
+(cloud-hosted) cluster and your existing Teleport Enterprise (self-hosted)
+cluster. You should be able to connect to both Teleport clusters and execute
+`tctl` commands using your current credentials.
+
+1. Log in to the self-hosted Teleport Enterprise cluster:
+
+   ```code
+   $ tsh login --proxy=<Var name="enterprise.example.com" /> --user=<Var name="myuser" />
+   $ tctl status
+   ```
+
+1. Log in to the cloud-hosted Teleport Enterprise cluster:
+
+   ```code
+   $ tsh login --proxy=<Var name="example.teleport.sh" /> --user=<Var name="myuser" />
+   $ tctl status
+   ```

```

Comment:
Might be worth adding SSO examples here, `--auth okta` or whatever (where `--user` will be ignored)